bitnami/consul/README.md

```diff
@@ -74,6 +74,7 @@ The following tables lists the configurable parameters of the HashiCorp Consul c
 | `securityContext.enabled`            | Enable security context                                          | `true`                                                     |
 | `securityContext.fsGroup`            | Group ID for the container                                       | `1001`                                                     |
 | `securityContext.runAsUser`          | User ID for the container                                        | `1001`                                                     |
+| `domain`                             | Kubernetes cluster domain                                        | `cluster.local`                                            |
```

Comment:
`domain` seems a very generic name to me, I would use `clusterDomain` as we use in other charts (redis for example)

https://github.com/bitnami/charts/blob/master/upstreamed/redis/values.yaml#L112-L114